src/mount/fuse/mount_config.cc

```diff
@@ -92,6 +93,7 @@ struct fuse_opt gSfsOptsStage2[] = {
 	SFS_OPT("bandwidthoveruse=%lf", bandwidthoveruse, 1),
 	SFS_OPT("sfsdirentrycachesize=%u", direntrycachesize, 0),
 	SFS_OPT("nostdmountoptions", nostdmountoptions, 1),
+	SFS_OPT("sfsuseoldwritealgorithm", useoldwritealgorithm, 1),
```

Comment:
Consider to match it here with the default value if relevant.

Reply:
I think it goes like this: if you parse the option, but don't find a value to assign to the parameter variable, then this last third parameter is the one assigned to the parameter variable. 

It was changed to the default value anyway, though now it must parse the value.